cmd/uhc/cluster/describe/describe.go

```diff
@@ -101,33 +124,59 @@ func run(cmd *cobra.Command, argv []string) {
 	}
 	defer connection.Close()
 
-	// Get the client for the resource that manages the collection of clusters:
-	resource := connection.ClustersMgmt().V1().Clusters()
-
-	// Get the resource that manages the cluster that we want to display:
-	clusterResource := resource.Cluster(argv[0])
-
-	// Retrieve the collection of clusters:
-	response, err := clusterResource.Get().
-		Send()
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "Can't retrieve clusters: %s", err)
-		os.Exit(1)
+	// Get full Api Response (JSON)
+	if !args.short {
+		// Get path
+		request := connection.Get().Path("/api/clusters_mgmt/v1/clusters/" + argv[0])
+
+		// Send the request:
+		responseGet, err := request.Send()
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Can't send request: %v\n", err)
+			os.Exit(1)
+		}
+
+		body := responseGet.Bytes()
+		check(err)
+		// Terminal or Json output
+		if !args.out {
+			err = dump.Pretty(os.Stdout, body)
+		} else {
+			myFile, err := os.Create("output.json")
```

Comment:
The filename should be read in from the args if a file is going to be created. Alternatively, `--output` could be changed to `--json` and the json could be dumped directly to stdout instead of a file.